persistence/selecting.md

```diff
@@ -29,18 +29,15 @@ graph TD
 start((start)) --> A{Using sagas<br /> or outbox?}
 A --> |Yes|B{Additional<br />requirements?}
```

Comment:
Were we going to use some other language here besides Additional Requirements?

Reply:
It's nasty, but I think it's out of scope of our current work.